malduck/extractor/extract_manager.py

```diff
@@ -291,6 +291,7 @@ def __init__(self, parent: ExtractManager) -> None:
         self.globals: Dict[str, Any] = {}
         self.parent = parent  #: Bound ExtractManager instance
         self.family = None  #: Matched family
+        self.files = {}
```

Comment:
`mypy` claims that it would be best to have it typed. You can add proper typing using TypedDict (https://mypy.readthedocs.io/en/stable/typed_dict.html)
```suggestion
        self.files = {}
```